operator/assembly.xml

```diff
@@ -0,0 +1,26 @@
+<assembly>
+
+    <id>operator</id>
+
+    <formats>
+        <format>zip</format>
+    </formats>
+
+    <includeBaseDirectory>true</includeBaseDirectory>
+
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/quarkus-app/</directory>
+            <outputDirectory></outputDirectory>
+            <includes>
+                <include>**/**</include>
+            </includes>
+            <excludes>
+            </excludes>
+        </fileSet>
+    </fileSets>
+
+    <files>
+    </files>
+
+</assembly>
```

Comment:
I think we don't need this anymore? Dockerfile copies jars directly, if I understand correctly.

Reply:
> The assembly archive is still present, but it will mostly be a product-only thing.
I think product should use the same Dockerfile, no? IIRC, the aim is to unify it.

Reply:
This isn't used now, correct. However, a zip of the operator jar and its Quarkus libraries is going to be needed for RHBK. The container build system is separate from the Java build system, so a container produced directly by maven plugin is not acceptable for product. The product operator Dockerfile will look much more like the server container's Dockerfile.

It's not possible to unify the processes as far as I can tell, because the quarkus maven plugin goes straight from populating the quarkus-app directory to building the container, with no chance to run another plugin, like the assembly plugin, in between.

We could put it in a profile if you want, so it's not part of the upstream build, but I think we're trying not to have a product profile this time